cloudbuild.yaml

```diff
@@ -3,92 +3,92 @@ steps:
+  args: ['tag', 'bazel/base:base_debian10', 'gcr.io/$PROJECT_ID/base-debian10:${COMMIT_SHA}']
 
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '--host_force_python=PY2', '//base:base-nonroot-debian9']
+  args: ['run', '--host_force_python=PY2', '//base:base-nonroot_debian9']
 - name: gcr.io/cloud-builders/docker
-  args: ['tag', 'bazel/base:base-nonroot-debian9', 'gcr.io/$PROJECT_ID/base-debian9:nonroot']
+  args: ['tag', 'bazel/base:base-nonroot_debian9', 'gcr.io/$PROJECT_ID/base-debian9:nonroot']
 - name: gcr.io/cloud-builders/docker
   args: ['tag', 'gcr.io/$PROJECT_ID/base-debian9:nonroot', 'gcr.io/$PROJECT_ID/base:nonroot']
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '--host_force_python=PY2', '//base:base-nonroot-debian10']
+  args: ['run', '--host_force_python=PY2', '//base:base-nonroot_debian10']
 - name: gcr.io/cloud-builders/docker
-  args: ['tag', 'bazel/base:base-nonroot-debian10', 'gcr.io/$PROJECT_ID/base-debian10:nonroot']
+  args: ['tag', 'bazel/base:base-nonroot_debian10', 'gcr.io/$PROJECT_ID/base-debian10:nonroot']
 
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '--host_force_python=PY2', '//base:debug-debian9']
+  args: ['run', '--host_force_python=PY2', '//base:debug_debian9']
 - name: gcr.io/cloud-builders/docker
-  args: ['tag', 'bazel/base:debug-debian9', 'gcr.io/$PROJECT_ID/base-debian9:debug']
+  args: ['tag', 'bazel/base:debug_debian9', 'gcr.io/$PROJECT_ID/base-debian9:debug']
 - name: gcr.io/cloud-builders/docker
   args: ['tag', 'gcr.io/$PROJECT_ID/base-debian9:debug', 'gcr.io/$PROJECT_ID/base:debug']
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '--host_force_python=PY2', '//base:debug-debian10']
+  args: ['run', '--host_force_python=PY2', '//base:debug_debian10']
 - name: gcr.io/cloud-builders/docker
-  args: ['tag', 'bazel/base:debug-debian10', 'gcr.io/$PROJECT_ID/base-debian10:debug']
+  args: ['tag', 'bazel/base:debug_debian10', 'gcr.io/$PROJECT_ID/base-debian10:debug']
 
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '--host_force_python=PY2', '//base:debug-nonroot-debian9']
+  args: ['run', '--host_force_python=PY2', '//base:debug-nonroot_debian9']
 - name: gcr.io/cloud-builders/docker
-  args: ['tag', 'bazel/base:debug-nonroot-debian9', 'gcr.io/$PROJECT_ID/base-debian9:debug-nonroot']
+  args: ['tag', 'bazel/base:debug-nonroot_debian9', 'gcr.io/$PROJECT_ID/base-debian9:debug-nonroot']
 - name: gcr.io/cloud-builders/docker
   args: ['tag', 'gcr.io/$PROJECT_ID/base-debian9:debug-nonroot', 'gcr.io/$PROJECT_ID/base:debug-nonroot']
 - name: gcr.io/cloud-builders/bazel
-  args: ['run', '--host_force_python=PY2', '//base:debug-nonroot-debian10']
+  args: ['run', '--host_force_python=PY2', '//base:debug-nonroot_debian10']
 - name: gcr.io/cloud-builders/docker
-  args: ['tag', 'bazel/base:debug-nonroot-debian10', 'gcr.io/$PROJECT_ID/base-debian10:debug-nonroot']
+  args: ['tag', 'bazel/base:debug-nonroot_debian10', 'gcr.io/$PROJECT_ID/base-debian10:debug-nonroot']
```

Comment:
These are tags, so they should use hyphens.

Reply:
Oh, but l think the tag names here are automatically derived from the Bazel targets. Then this makes sense, I guess?

Reply:
That is my understanding: The bazel docker rules automatically generate those tag names; I'm assuming `cloudbuild.yaml` is what publishes the production packages, so the final `gcr.io` tag names are unchanged. I hope?